docs/source/guide/webhooks.md

```diff
@@ -0,0 +1,138 @@
+  </tr>
+  <tr>
+    <td>Annotation Created</td>
+    <td>For a specific project, triggers when new annotations are created for any tasks. One event per annotation.</td>
+    <td>Use to start training in an active learning scenario.</td>
+  </tr>
+  <tr>
+    <td>Annotation Updated</td>
+    <td>For a specific project, triggers when an existing annotation is updated, overwritten, or when a task is skipped.</td>
+    <td>Use to prompt model retraining. </td>
+  </tr>
+  <tr>
+    <td>Annotations Deleted</td>
+    <td>For a specific project, triggers when an annotation is deleted.</td>
+    <td>Use to create a new version of a training dataset. </td>
+  </tr>
+  <tr>
+    <td>Project Created</td>
+    <td>For an organization, triggers when a project is created.</td>
+    <td>Use to create a new pipeline for data management.</td>
+  </tr>
+  <tr>
+    <td>Project Updated</td>
+    <td>For an organization, triggers when project settings are updated, created, or saved.</td>
+    <td>Use to update an existing data management pipeline.</td>
+  </tr>
+  <tr>
+    <td>Project Deleted</td>
+    <td>For an organization, triggers when a project is deleted.</td>
+    <td>Use to remove a data management pipeline. </td>
+  </tr>
+</table>
+
+## How to integrate with webhooks
+
+To integrate with webhooks in Label Studio, you must do the following:
+1. Create a webhook endpoint or URL as part of your application or machine learning model pipeline.
+2. Make sure the webhook endpoint or URL is accessible to your Label Studio instance,  can accept HTTP POST requests.
+3. [Add the webhook to Label Studio](webhooks.html#Add-a-new-webhook-in-Label-Studio).
+
+Set up your webhook endpoint to read the event payload from Label Studio and take action based on the event.
+
+## Add a new webhook in Label Studio
+
+Set up a webhook URL in Label Studio and associate it with one or more event triggers. You can set up a webhook URL in the Label Studio UI, or using the API. You can configure as many webhook connections as you want, but too many might overwhelm your instance. 
+
+### Add a webhook in Label Studio UI
+
+Add a webhook URL to Label Studio. The webhook URL must be set up to accept HTTP POST requests.
+
+1. In the Label Studio UI, open the project that you want to associate with a webhook URL.
+2. Click **Settings** and click **Webhooks**.
+3. Click **New Webhook**. 
+4. In the **Payload URL** field, provide the URL to send event payloads to. 
+5. (Optional) Toggle the **Is Active** option to deactivate the webhook until it is ready to use. Otherwise, the webhook becomes active as soon as you save it and events are sent to the URL.
+6. (Optional) Add any headers required by the webhook URL. Specify the header name and the value. MUST BE AN ALPHANUMERIC STRING, CAN'T ADD MORE THAN 10. You can use headers to authenticate a request to your webhook URL.
+7. Select whether to send a payload with the event. HERE'S WHAT HAPPENS IF YOU DON'T SEND A PAYLOAD.
```

Comment:
What happens if you don't send a payload, @chiganov ? Just a POST with the name of the event?

Reply:
Yes, only `action` key